src/dynarec/la64/dynarec_la64_avx_66_0f38.c

```diff
@@ -200,7 +200,7 @@ uintptr_t dynarec64_AVX_66_0F38(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t i
         case 0x2E:
             INST_NAME("VMASKMOVPS Ex, Gx, Vx");
```

Comment:
Are you sure it's `Ex, Gx, Vx`, in this order? 

... nope, it follows this table:

| Op/En | Operand 1     | Operand 2    | Operand 3     | Operand 4 |
| ----- | ------------- | ------------ | ------------- | --------- |
| RVM   | ModRM:reg (w) | VEX.vvvv (r) | ModRM:r/m (r) | N/A       |
| MVR   | ModRM:r/m (w) | VEX.vvvv (r) | ModRM:reg (r) | N/A       |

Oh I saw the Arm64 side (@ptitSeb) has this weird order too, I think it's a mistake.

Reply:
Just to be clear, the comment / INST_NAME might be confusiong / incorrect, but the code is exact. Mask is VX, EX/GX are destination/source, so ARM64 code is correct.

Most game that use AVX use this opcode (like satisfactory for example), it's been tested quite a lot already.